styles/mixins/patterns/_containers.scss

```
@@ -235,3 +235,26 @@ $-header-height: var.$touch-target-size;
 @mixin scrollbox--with-header {
   @include scrollbox($shadow-top-position: $-header-height);
 }
+
+/**
+ * A sticky container that is sized one-touch-unit high. This is currently
+ * expanded upon by table styling, but not used in other patterns (yet).
+ */
+@mixin sticky-header {
```

```suggestion
@mixin sticky-scrollbox-header {
```

The association of the two names could help to understand when to use the pattern.

---

You're right to note that this pattern isn't quite normalized where we need it yet. I spent a few minutes trying to see if I could re-jigger it (also taking the comment below about color into consideration), but the pattern lacks clarity out of context. What I'd like to propose is that we carry this forward into the next set of work, which involves a `Table` component that uses the `Scrollbox` component as a wrapper. Then we can scrutinize this pattern in better context and make some improvements.